example/src/main/java/io/netty/example/qotm/QuoteOfTheMomentClient.java

```diff
@@ -52,7 +53,7 @@ public static void main(String[] args) throws Exception {
             // Broadcast the QOTM request to port 8080.
             ch.writeAndFlush(new DatagramPacket(
                     Unpooled.copiedBuffer("QOTM?", CharsetUtil.UTF_8),
-                    new InetSocketAddress("255.255.255.255", PORT))).sync();
+                    SocketUtils.socketAddress("255.255.255.255", PORT))).sync();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this IP "255.255.255.255" address configurable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1313)